packages/core/src/index.ts

```diff
@@ -187,4 +192,34 @@ export class JanusNameService {
       throw new Error(result.Errors[0].toString());
     }
   }
+
+  public async RenewDomain(
+    domain: string,
+    tld: string
+  ): Promise<RequestResult> {
+    domain = domain.toLocaleLowerCase();
+    tld = tld.toLocaleLowerCase();
+
+    const request = new RenewDomainRequest(domain, tld);
+
+    const config = Bootstrapper.Resolve<NameServiceConfig>('NameServiceConfig');
+
+    const validator = new RenewDomainValidator(config);
+    const validation = await validator.ValidateRenewDomainRequest(request);
+
+    const result = new RequestResult();
+    result.Success = validation.isValid();
+    result.Errors = validation.getFailureMessages();
+
+    if (result.Success) {
+      try {
+        const dealed = await this._jnsService.RenewDomain(request);
+        return dealed;
+      } catch (error) {
+        throw new Error(error);
+      }
+    } else {
+      throw new Error(result.Errors[0].toString());
```

Comment:
This `.toString()` here is redundant, because the type of `result.Errors` is `string[]`.